docs/tutorials/vertical-fl.md

```diff
@@ -67,7 +67,7 @@ This can all be performed with ease using a data provisioning pipeline. To run i
 :warning: Proceed to the next step only once the pipeline completes. This pipeline will create data in 3 distinct locations.
 
 ## Model preparation for VFL
-It is an ongoing research topic on how the model can be orchestrated in VFL. We have decided to go with the most common approach by splitting it between the host and contributors, also referred to as **split learning**, this approach can be easily altered by moving layers between parties to hosting whole model on contributors while host provides only aggregation and/or activation function. We believe that this can better demonstrate capabilities of VFL on AzureML and most of the existing models can be easily split without requiring too much work.
+It is an ongoing research topic on how the model can be orchestrated in VFL. We have decided to make the most common approach as default by splitting it between the host and contributors, also referred to as **split learning**, this approach can be easily altered by moving layers between parties to hosting whole model on contributors while host provides only aggregation and/or activation function. We believe that this can better demonstrate capabilities of VFL on AzureML and most of the existing models can be easily split without requiring too much work. However, **CCFRAUD** example is also runnable with unsupervised latent representation learning through Variational Autoencoder (VAE), which is shared once with the host and thus significantly reduces communication overhead. The host then uses these latent representations to predict the target values. More about this approach, and how to set it up, can be read in the section [below](#single-shot-learning-through-variational-autoencoder).
```

Comment:
For educational value, I'm beginning to think we might need to split this into two distinct pipelines?

Reply:
The tutorial itself starts to become something very configurable, maybe too much.

Reply:
We can definitely open this topic during the deep dive tomorrow. The only change in the pipeline itself, as it goes for the user, is the name of the network being trained `SimplerLinear` -> `SimpleVAE`, so the **number of configurations does not change**. The downside I see here is that the approach is a bit hidden and having separate pipeline may help it to stand out?